applications/sintering/include/pf-applications/sintering/operator_base.h

```diff
@@ -512,7 +512,7 @@ namespace Sintering
     }
 
     void
-    initialize_system_matrix() const
+    initialize_system_matrix(const bool print_stats = true) const
```

Comment:
Why is there no new parameter in the parameter struct?

Reply:
This is just to make it simpler. I thought that this option is not so crucial. Mainly, I decided to add it to get rid of the non-relevant output in tests.